eth/backend.go

```diff
@@ -162,6 +161,7 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 		bloomRequests:     make(chan chan *bloombits.Retrieval),
 		bloomIndexer:      core.NewBloomIndexer(chainDb, params.BloomBitsBlocks, params.BloomConfirms),
 		p2pServer:         stack.Server(),
+		discmix:           enode.NewFairMix(2 * time.Second),
```

Comment:
Just curious how you came up with this? Some other fair mixes are 1 second or 5 second timeouts.

Reply:
I just set it by gut feel, and it's kind of wrong actually.

The implication of the timeout is: if the filtered discv5 stream fails to come up with a node within 2s, we will give up on balancing DNS with the DHT and just pull one from DNS. The DNS list always has nodes and will likely be quicker than 2s. The timeout is per draw, so if the DHT doesn't have any matching nodes, we can produce at most one node every 2s.

How about we go with 100ms?

Reply:
So in this particular case, we can think of the timeout as the 'head start' the DHT has over DNS. As we don't expect the DHT to have any EL nodes right now, we might as well set it to zero to have the same semantics as before.

Reply:
Not sure if I have enough intuition to say. At 100ms won't we end up more heavily weighted towards DNS list? Or more draws gives a better chance for getting DHT nodes?

Reply:
Crossed messages :) The main thing to understand here is that any timeout will increase the discovery latency. Since discovery efficiency is already quite bad (~1%) we should probably not slow it down even more, is what I mean. I don't particularly care about fairness between DHT and DNS here.

Reply:
Okay that makes sense to me.